src/util/addresses.ts

```diff
@@ -44,15 +54,14 @@ export const UNISWAP_MULTICALL_ADDRESSES: AddressMap = {
   ...constructSameAddressMap('0x1F98415757620B543A52E61c46B32eB19261F984'),
   [ChainId.CELO]: CELO_MULTICALL_ADDRESS,
   [ChainId.CELO_ALFAJORES]: CELO_MULTICALL_ADDRESS,
-
   [ChainId.ARBITRUM_GOERLI]: ARBITRUM_GOERLI_MULTICALL_ADDRESS,
+  [ChainId.BSC]: BSC_MULTICALL_ADDRESS,
   // TODO: Gnosis + Moonbeam contracts to be deployed
 };
 
 export const OVM_GASPRICE_ADDRESS =
   '0x420000000000000000000000000000000000000F';
 export const ARB_GASINFO_ADDRESS = '0x000000000000000000000000000000000000006C';
-
 export const TICK_LENS_ADDRESS = '0xbfd8137f7d1516D3ea5cA83523914859ec47F573';
 export const NONFUNGIBLE_POSITION_MANAGER_ADDRESS =
   '0xC36442b4a4522E871399CD717aBDD847Ab11FE88';
```

Comment:
Since swap router02 is deployed at a different address on BSC (0xa64Dd3A63fEed608660226112E748F93B87E650F) than the canonical one below, we need to create an address map and update `util/methodParameters.ts` to use the new mapping